simulation/drequirements.txt

```diff
@@ -0,0 +1,5 @@
+eth-utils
```

Comment:
I wrote to you that it was unnecessary, you said it was just left since you solving the problem. If this is not necessary, it is better to remove.